index.html

```diff
@@ -5,8 +5,40 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="main.css">
 </head>
 <body>
+    <nav class="container">
+        <div class="logo"><span class="bx-sh"> Icarus </span>  UI</div>
+        <div class="a">
+            <a href="#" class="a1">Home</a>
+            <a  href="#"  class="a1">Docs</a>
+        </div>
+    </nav>
     
+    <main>
+        <section class="main-content">
+            <h1><span class="bx-sh">Icarus UI</span></h1>
+            <p>
+                It is the
+                <span class="un-txt">" Light weight "</span> CSS library to develop
+                <span class="un-txt">" Responsive "</span> web apps.
+            </p>
+            <div>
+                <a href="#">
+                    <button class="primary">Get started</button>
+                </a>
+
+                <a href="#">
+                    <button class="secondary">Github</button>
+                </a>
```

Comment:
You should not put a button inside an `a` tag. If the purpose is to navigate the user to a different page then use the `a` tag directly or else use a `button`. If you want the `a` tag to look like a button then use the same class to style both `button` and `a` tag.